app/src/main/java/org/geekhaven/ishare/activities/ResetPasswordActivity.kt

```diff
@@ -0,0 +1,62 @@
+package org.geekhaven.ishare.activities
+
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.text.TextUtils
+import android.view.View
+import android.widget.Button
+import android.widget.EditText
+import android.widget.ProgressBar
+import android.widget.Toast
+
+import com.google.android.gms.tasks.OnCompleteListener
+import com.google.android.gms.tasks.Task
+import com.google.firebase.auth.FirebaseAuth
+
+import org.geekhaven.ishare.R
+
+class ResetPasswordActivity : AppCompatActivity() {
+
+    private var inputEmail: EditText? = null
+    private var btnReset: Button? = null
+    private var btnBack: Button? = null
+    private var auth: FirebaseAuth? = null
+    private var progressBar: ProgressBar? = null
```

Comment:
Prefer to make your variable as val and local, if possible.
If not possible try if your variable can be global and lateinit var.